src/esp/bindings/AttributesManagersBindings.cpp

```diff
@@ -274,12 +274,14 @@ void initAttributesManagersBindings(py::module& m) {
 
   // ==== Light Layout Attributes Template manager ====
   declareBaseAttributesManager<LightLayoutAttributes>(m, "BaseLightLayout");
+  // NOLINTNEXTLINE(bugprone-unused-raii)
```

Comment:
Would like some confirmations from the reviewers that these are false positives.

Reply:
Yeah, those are all false positives.  We should probably turn this off globally for src/esp/bindings.  The most common use of RAII is locking mutexs and there likely won't ever be any mutexs in bindings.

Reply:
Unfortunately, disabling a single check from clang-tidy in a subfolder is rather difficult unless we upgrade to clang-tidy 11 which supports inheriting from the parent config. As such, I'll just leave them as warnings.